specifyweb/stored_queries/queryfieldspec.py

```diff
@@ -22,7 +22,7 @@
 DATE_PART_RE = re.compile(r'(.*)((NumericDay)|(NumericMonth)|(NumericYear))$')
 
 # Pull out author or groupnumber field from taxon query fields.
-TAXON_FIELD_RE = re.compile(r'(.*) ((Author)|(GroupNumber))$')
+TAXON_FIELD_RE = re.compile(r'(.*) ((Author)|(groupNumber))$')
```

Comment:
FYI: from what Ben told me, the only reason we limit what fields are accessible in tree queries is because that's what sp6 did
that reason might be less good today compared to how it was in 2013.
@realVinayak interested in tackling this at some point?

Reply:
See https://github.com/specify/specify7/pull/4929. Begun some work there. now need to add tests to ensure compatibility with 6.